components/kyma-environment-broker/internal/storage/driver/postsql/instance.go

```diff
@@ -345,6 +345,10 @@ func (s *Instance) toInstance(dto dbmodel.InstanceDTO) (internal.Instance, error
 	if err != nil {
 		return internal.Instance{}, errors.Wrap(err, "while decrypting parameters")
 	}
+	err = s.cipher.DecryptKubeconfig(&params)
```

Comment:
provisioning parameters are stored also in the operations - so you we have to encrypt kubeconfig there

Reply:
Got it.

Reply:
Added.